ruby/hangman/hangman.rb

```diff
@@ -18,7 +17,7 @@ def is_game_over
 	end
 
 	def add_guess_count
-		if @solution.index(@guess) == nil && was_guess_repeat == true
+		if @solution.index(@guess) == nil && was_guess_repeat != true
```

Comment:
when working with code like this that does not require an else try using one line syntax.

`counter += 1 if (@solution.index(@guess) == nil) && (was_guess_repeat != true)`